test cases/common/51 pkgconfig-gen/meson.build

```diff
@@ -19,7 +19,7 @@ pkgg.generate(
 )
 
 pkgconfig = find_program('pkg-config', required: false)
-if pkgconfig.found()
+if pkgconfig.found() and build_machine.system() != 'windows'
```

Comment:
That doesn't skip the whole test though, only the validation of the pkg-config file. I've instead added that to the `113 generatorcustom` test which is indeed wholly skipped with the VS backend.